crypto/cms/cms_env.c

```diff
@@ -55,6 +83,25 @@ static CMS_EnvelopedData *cms_enveloped_data_init(CMS_ContentInfo *cms)
     return cms_get0_enveloped(cms);
 }
 
+static CMS_AuthEnvelopedData *
+cms_auth_enveloped_data_init(CMS_ContentInfo *cms)
+{
+    if (cms->d.other == NULL) {
+        cms->d.authEnvelopedData = M_ASN1_new_of(CMS_AuthEnvelopedData);
+        if (cms->d.authEnvelopedData == NULL) {
+            CMSerr(0, ERR_R_MALLOC_FAILURE);
+            return NULL;
+        }
+        cms->d.authEnvelopedData->version = 0;
```

Comment:
Put a ref to documentation also to where this version  = 0 comes from

Reply:
I mean a reference to the section in the RFC here also..

Reply:
As an example of somewhere where I have done this for code..:
```
/*
 * Generate a SP800-56B RSA key.
 *
 * See SP800-56Br1 6.3.1 "RSA Key-Pair Generation with a Fixed Public Exponent"
 *    6.3.1.1 rsakpg1 - basic
 *    6.3.1.3 rsakpg1 - crt
```

Reply:
Ok this is defined as part for the type so added ref to RFC and section. Please let me know if the format is fine.